### ISSUE TITLE
Usage experience

### ISSUE THREAD

**Issue by micksh6** [thegouger/MIDI2LR#27]
Don't know where to post this, I hope it's the right place, even though I have no issue to report.

I bought Korg NanoKontrol2 especially for this because I don't like mouse control in Lightroom. I tried to improve mouse in LR, bought gamer's high resolution mouse, with button switching sensitivity, but still, couldn't adjust sliders quickly and precise enough.

This software promised much easier and more natural controls and it does deliver up to promises. Author did a great job!
Apart from need for more refined Wihite Balance controls (already addressed here https://github.com/thegouger/MIDI2LR/issues/25 ) the software works very well.

I processed several photos with MIDI2LR and Korg NK2. It works, and I'm sure it will allow faster adjustments than mouse/keyboard.

The major problem for me is to get used to this software/hardware combination on physical level - develop reflexes, etc. The habit of using mouse is too strong.

To break the habit I made a cheat sheet with functions printed over NK2 control layout. This is what I'm starting with, I'm sure others will choose different control mapping. This is to show a basic idea of how to get used to new tools, and maybe to inspire discussion about other ideas.
Printed this picture: http://1drv.ms/1U7FKWu
Setting file just in case: http://1drv.ms/1U7FTZQ

So, mostly, the control order on faders is the same as in LR, starting from exposure. Faders are bettern than knobs IMO. 
White balance is moved to knobs because I rarely need to adjust it (Auto WB works well on my camera, plus MIDI2LR needs finer granularity). Then I assigned the rest 6 knobs to whatever I think are the most used functions.

I'm glad buttons made assignable recently (Thanks thegouger!). It would be nice if more functions could be assigned, like Copy/paste settings. Currently I use extra mouse buttons for these. 
And there are other weird 3-key LR shortcuts that beg to be assigned to single button.

To summarize, thanks a lot to author for great software; and all ideas on how to break mouse habit and migrate to this smoothly would be appreciated.

**Comment by rsjaffe** [comment]
This issue was moved to rsjaffe/MIDI2LR#11
